x-pack/osquerybeat/ext/osquery-extension/specs/elastic_ntfs_file.yaml

```
@@ -0,0 +1,115 @@
+type: table
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)